HelloSignalR/Providers/AuthenticationProvider.cs

```diff
@@ -0,0 +1,35 @@
+﻿using AspNet.Security.OpenIdConnect.Server;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Authentication.OpenIdConnect;
+using AspNet.Security.OpenIdConnect.Extensions;
+
+namespace HelloSignalR.Providers
+{
+    public class AuthenticationProvider : OpenIdConnectServerProvider
+    {
+        public override Task ValidateClientAuthentication(ValidateClientAuthenticationContext context)
+        {
+            context.Skipped();
```

Comment:
Please consider adding a check here to ensure the client_id is the intended one. A comment indicating why `Skipped` is called might also be worth it.